### DISCUSSION TITLE
App is not executing when "import folium" is included in the main script.

### DISCUSSION THREAD

**Discussion by jankovicmmarko18** [pyinstaller/pyinstaller#5875] [Help]
## Description of the issue
I am trying to make an executable app with folium package and it gives an error no matter what i try. I've tried this solution: https://github.com/pyinstaller/pyinstaller/issues/5256 but it doesn't work on my side.  I've also tried "--hidden-import branca --additional-hooks-dir C:\Users\janko\anaconda3\envs\gis\Lib\site-packages\branca" but that also gives the same error when i try to execute the compiled app.
BTW app works perfectly when folium is not included. 

### Context information (for bug reports)

* Output of `pyinstaller --version`: 4.1
* Version of Python: 3.8
* Platform: Windows 10
* How you installed Python: conda
* Did you also try this on another platform? Does it work there? No I didn't try. 

### A minimal example program which shows the error

```
File "branca\colormap.py", line 20, in <module>
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\janko\\AppData\\Local\\Temp\\_MEI85162\\branca\\_cnames.json'
[6460] Failed to execute script C2F_Alter_Engine_v1.4
```

**Comment by rokm** [comment]
> --additional-hooks-dir C:\Users\janko\anaconda3\envs\gis\Lib\site-packages\branca

Additional hooks dir should point to the location containing the hook for branca, not the branca package itself (unless you put the hook there).

And you'll also need to collect data for `folium` using a custom hook that does `collect_data_files('folium')`.

If you were using PyInstaller 4.3, you could also just add `--collect-data branca --collect-data folium` to your PyInstaller command-line instead of having to write hooks.
